crates/miden-testing/src/kernel_tests/block/proposed_block_success.rs

```diff
@@ -272,18 +263,23 @@ fn noop_tx_and_state_updating_tx_against_same_account_in_same_block() -> anyhow:
         .with_component(MockAccountComponent::with_empty_slots());
 
     let mut builder = MockChain::builder();
-
     let mut account0 = builder.add_account_from_builder(
         Auth::Conditional,
         account_builder,
         AccountState::Exists,
     )?;
 
+    let noop_note0 =
+        NoteBuilder::new(ACCOUNT_ID_SENDER.try_into().unwrap(), &mut rand::rng()).build()?;
+    let noop_note1 =
+        NoteBuilder::new(ACCOUNT_ID_SENDER.try_into().unwrap(), &mut rand::rng()).build()?;
+    builder.add_note(OutputNote::Full(noop_note0.clone()));
+    builder.add_note(OutputNote::Full(noop_note1.clone()));
```

Comment:
For a follow-up PR, I'd make the name of `builder.add_note` more explicit.
Just by reading this, I'm not sure what exactly is happening: is there going to be a transaction created by the mock builder that consumes these notes? Or are these notes going to appear as output out of thin air?

IIUC, it's the latter, in which case I'd maybe rename to `add_output_note`